docs/source/onnx.rst

```diff
@@ -342,9 +342,15 @@ If the operator is an ATen operator (shows up in the TorchScript graph with the
   Make sure the function has the same name as the ATen function, which may be declared in
   ``torch/_C/_VariableFunctions.pyi`` or ``torch/nn/functional.pyi`` (these files are generated at
   build time, so will not appear in your checkout until you build PyTorch).
-* The first arg is always the ONNX graph that is being built for export.
+* The first arg is always the ONNX graph that is being built for export (except for advanced symbolic functions).
   Other arg names must EXACTLY match the names in the ``.pyi`` file,
   because dispatch is done with keyword arguments.
+* Advanced symbolic functions may accept an additional `torch.onnx.SymbolicContext`
```

Comment:
Suggested wording:

A symbolic function that has a first arg (before the ``Graph`` object) with the type annotation of ``torch.onnx.SymbolicContext`` will be called with that additional context. See examples below.